src/transformers/models/whisper/english.json

```diff
@@ -0,0 +1,1742 @@
+{
```

Comment:
I would probably add this to the `tokenizer_config.json` file instead: https://huggingface.co/openai/whisper-medium.en/blob/main/tokenizer_config.json

Reply:
It seems model specific so it should be in the checkpoint repos yes. Fine with me if it's in the tokenizer config!

Reply:
It's more of a utility, as it could be used by any model. Normalising multilingual outputs that where translated is pretty useful! 
So two solution : 
1. We add it to the tokenizer_config.json 
2. Add it to the `english_normaliser.py` file as any model can use it. 
I am in favor of 2 -> needs less changes and is more convenient

Reply:
This would work for me as well. I image you want to add it as a simple hardcoded dict:

ENGLISH_DISFLUENCIES = {

}

?

I could see this dictionary changing over time and being different for different models (which would be an argument to put it on the Hub), but happy to leave it "hard-coded" directly in `english_normaliser.py` for now as we can always move it to the Hub without breaking changes down the road.

So 2.) is also good for me (a bit indifferent between the two)

Reply:
Given the size of it I don't really want it in any files of the library. So if not in the checkpoint repos, it should be in a dataset (like we do for ImageNet labels or Coco labels)

Reply:
This adds the dataset dependency, but I guess it is not really a problem

Reply:
It's already a soft dep, so not a problem at all!

Reply:
No this should not be in a datasets library here. Let's put it in the checkpoints then. One doesn't need `datasets` for inference for whisper at the moment and I would like to have it stay this way.